### PR TITLE
HARVESTER: Add addon

### PR DESCRIPTION
**Pull request by n313893254**
### Summary
image

image

Fixes #

- https://github.com/harvester/harvester/issues/2845
- https://github.com/harvester/harvester/issues/2913

### Areas or cases that should be tested

test step:
1.  You can turn on and off vm-import and pci's addon in the UI
2. If pci addon is disabled, the PCI device list page only displays a message that jumps to the addon setting page.
3. If addon is enabled, then you should see the corresponding deployment in the harvester-system namespace
image

### CODE REVIEW

**Comment by WuJun2016** [comment]
@n313893254   PTAL, thanks

**Review by n313893254 — COMMENTED** [review]
LGTM

File: pkg/harvester/l10n/en-us.yaml [hunk 1]
```diff
@@ -186,6 +186,10 @@ harvester:
     enableGroup: Enable Group
     disableGroup: Disable Group
     labelRequired: "This rule should not be manually altered: it ensures that the PCI devices selected for this virtual machine are available on the VM's host."
+    goSetting:
+      prefix: pcidevices-controller addon is not enabled, click
+      middle: here
+      suffix: to enable it to manage your PCI devices.
```
[the remark this answers is not in the document]
Comment (WuJun2016) [reply pkg/harvester/l10n/en-us.yaml:192]: I remember there were some problems with this, such as the page being refreshed when clicked,

[the remark this answers is not in the document]
Comment (n313893254) [reply pkg/harvester/l10n/en-us.yaml:192]: Yep, I think this PR can be merged, let me investigate it.

**Pull request merged**